dotcom-rendering/src/web/lib/LiveBlogRenderer.tsx

```diff
@@ -23,6 +26,21 @@ export const LiveBlogRenderer = ({
 }: Props) => {
 	return (
 		<>
+			{pinnedPost && (
```

Comment:
Where you've put the `Island` above does feel like a good location but it could also live here alongside the post itself?

Reply:
Good call, I've moved the island within the pinnedpost conditional check so its next to the pinned post.